README.md

```diff
@@ -51,49 +51,66 @@ env_var.sh
 #### Run the translator script (from Work_Zone)
 
 ```
-python -m translator.source_code.icone_translator -i inputfile.xml -o outputfile.geojson
+python -m translator.icone_translator inputfile.xml --outputFile outputfile.geojson
 ```
 
 Example usage:
 
 ```
-python -m translator.source_code.icone_translator -i 'translator/sample files/icone data/incidents_extended.xml' 
+python -m translator.icone_translator 'translator/sample files/icone data/incidents_extended.xml' 
 ```
 
 ### Execution for COtrip translator
 
-#### Run the translator script (from Work_Zone) and Please specify a valid Json file as an input file !
+#### Run the translator script (from Work_Zone)
+
+```
+python -m translator.cotrip_translator inputfile.json --outputFile outputfile.geojson
+```
+
+Example usage:
+
+```
+python -m translator.cotrip_translator 'translator/sample files/cotrip_data/cotrip_1.json'
+```
+
+### Execution for NavJoy 568 translator
+This translator reads in a NavJoy 568 speed reduction form an translates it into a WZDx message. Many of the 568 messages cover 2 directions of traffic, and are thus expanded into 2 WZDx messags, one for each direction. 
```

Comment:
```suggestion
This translator reads in a NavJoy 568 speed reduction form and translates it into a WZDx message. Many of the 568 messages cover 2 directions of traffic, and are thus expanded into 2 WZDx messags, one for each direction. 
```

Reply:
apparently I need to use spellcheck more. Sorry about silly errors like this

Reply:
fixed